tests/ci-run-tests.sh

```diff
@@ -6,12 +6,12 @@ for x in $(ls test*.py */test*.py); do
   echo "Testing $x:" 
   echo 
   if [ $x == "ensdf_processing.py" ]; then
-    nosetests test_ensdf_processing.py --process-timeout=120
+    pytest test_ensdf_processing.py --process-timeout=120
   else
-    nosetests "$x"
+    pytest "$x"
   fi
   status=$?
-  if [ $status -ne 0 ]; then 
+  if [ $status -ne 0 ] && [ $status -ne 5 ]; then 
```

Comment:
Exit code  `5` is given if no tests are collected. I added it because `test_dtypes.py` has no tests in it causing tests after this file to stop running. I am still determining the purpose of this file. We might consider removing this file or populating it with tests.